### PR TITLE
Adds ability to run scripts in ocm-container

### PR DESCRIPTION
**Pull request by iamkirkbater**
Adds the ability to run a script on a cluster using ocm-container,
enabling looping through multiple clusters to run the same script when
necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,3 +141,31 @@ On a Mac, it seems that it doesn't follow the default kinit functionality where
 We've built in functionality to simplify the cluster login steps.  Now within the contianer you can run `sre-login cluster-id` and it will refresh your ocm login, create a tunnel within the container if necessary, and then log-in to the cluster.
 
 `sre-login` accepts both a cluster-name or a cluster-id.  If the cluster-name is not unique, it will not ask which one, but display the clusters and exit.
+
+### Advanced scripting with ocm-container
+We've recently added the ability to run a script within the container so that you can run ocm-container within a script.
+
+Given the following shell script saved on the local machine in `~/myproject/script.sh`:
```
Comment (georgettica) [inline README.md:148]:
```suggestion
Given the following shell script saved on the local machine in `~/myproject/in-container.sh`:
```

File: README.md [hunk 2]
```diff
@@ -141,3 +141,31 @@ On a Mac, it seems that it doesn't follow the default kinit functionality where
...
+```
+#!/bin/bash
```
Comment (georgettica) [inline README.md:150]: to make it clear this is the script that is in the container
```suggestion
cat ~/myproject/in-container.sh
#!/bin/bash
```

File: README.md [hunk 3]
```diff
@@ -141,3 +141,31 @@ On a Mac, it seems that it doesn't follow the default kinit functionality where
...
+
+# source this so we get all of the goodness of ocm-container
+source /root/.bashrc
+
+# get the version of the cluster
+oc version >> report.txt
+```
+
+We can run that on-container with the following script:
```
Comment (georgettica) [inline README.md:159]:
```suggestion
We can run that on-container with the following script which runs on the host (~/myproject/on-host.sh):
```

File: README.md [hunk 4]
```diff
@@ -141,3 +141,31 @@ On a Mac, it seems that it doesn't follow the default kinit functionality where
...
+```
+#!/bin/bash
```
Comment (georgettica) [inline README.md:161]:
```suggestion
cat ~/myproject/on-host.sh
#!/bin/bash
```

File: README.md [hunk 5]
```diff
@@ -141,3 +141,31 @@ On a Mac, it seems that it doesn't follow the default kinit functionality where
 We've built in functionality to simplify the cluster login steps.  Now within the contianer you can run `sre-login cluster-id` and it will refresh your ocm login, create a tunnel within the container if necessary, and then log-in to the cluster.
 
 `sre-login` accepts both a cluster-name or a cluster-id.  If the cluster-name is not unique, it will not ask which one, but display the clusters and exit.
+
+### Advanced scripting with ocm-container
+We've recently added the ability to run a script within the container so that you can run ocm-container within a script.
```
Comment (georgettica) [inline README.md:146]: not sure if we would revise this in the future, but the wording of `we've recently added` would not make sense after  a while

**Review by georgettica — COMMENTED** [review]
basically I had a tiny difficulty reading the two blocks and thinking they were the same script

added suggestion blocks to make it more clear for me at least, if you think it's too much feel free to disregard

all in all, a required feature and saves much toil of doing operations in ocm-container

**Comment by iamkirkbater** [comment]
I wanted to wait until after @clcollins' #20 was merged before merging this.  The LGTM label was removed because I fixed a README merge conflict, but I didn't want to create any merge conflicts in #20 so I waited until that was merged before merging this one.

**Pull request merged**